### ISSUE TITLE
[虫子]：

### ISSUE THREAD

**Issue by Hawkeww** [Yimura/YimMenu#1536]
### Describe the bug

Request to restore the menu layout. The new layout is very inconvenient! Please return to the previous layout! Please restore the layout of personal vehicles and network options, as well as the menu ui color.

### Steps To Reproduce

Request to restore the menu layout. The new layout is very inconvenient! Please return to the previous layout! Please restore the layout of personal vehicles and network options, as well as the menu ui color.

### Expected Behavior

Request to restore the menu layout. The new layout is very inconvenient! Please return to the previous layout! Please restore the layout of personal vehicles and network options, as well as the menu ui color.

### Screenshots and/or Logs

```
<paste your logs here>
```

### Store

Steam

### Additional context

Request to restore the menu layout. The new layout is very inconvenient! Please return to the previous layout! Please restore the layout of personal vehicles and network options, as well as the menu ui color.

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

**Comment by Tayipc38** [comment]
The ui colors can be changed in the settings tab. And no the overlay changes will not be reverted to that chaos.

**Comment by tupoy-ya** [comment]
~~Let's revert #62 the new layout is awful!!~~

The colors are bad, we don't even change the button color when it's hovered now.
`void gui::push_theme_colors()`
```cpp
auto button_hovered_color =
	ImVec4(button_color.x + 0.14f, button_color.y + 0.14f, button_color.z + 0.14f, button_color.w);
ImGui::PushStyleColor(ImGuiCol_ButtonHovered, button_hovered_color);
```

**Comment by Tayipc38** [comment]
> ~Let's revert #62 the new layout is awful!!~
> 
> The colors are bad, we don't even change the button color when it's hovered now. `void gui::push_theme_colors()`
> 
> ```c++
> auto button_hovered_color =
> 	ImVec4(button_color.x + 0.14f, button_color.y + 0.14f, button_color.z + 0.14f, button_color.w);
> ImGui::PushStyleColor(ImGuiCol_ButtonHovered, button_hovered_color);
> ```

The current gui settings can be expanded by automatically adjusting the alpha value (preferably like 0.2 higher) for the hovered and active states of elements.

**Comment by ghost** [comment]
The new gui has certain problems, for example the player section has internal sliders so not only are they both vertical and horizontal like in the "toxic" section (for example, to set the wanted level you need to move horizontally). Is it possible to just stretch the entire player section (that, I think, other sections too) horizontally and then you don’t have to move the slider up and to the right to avoid unnecessary movements? This also applies to the "Network" section (also make internal elements in 2-3 columns so that the slider disappears) and the players' database section when the lower save and delete buttons are not visible due to the slider and simply place them on the side. I understand that this can slightly reduce the overview of the game itself, so it will be much more convenient. Or is it a problem with my 125x FHD laptop resolution?
![pic2](https://github.com/YimMenu/YimMenu/assets/136161998/76824b5a-bdcf-4790-a6ce-8fb87366c7f2)
![pic1](https://github.com/YimMenu/YimMenu/assets/136161998/32e595de-5485-49b3-ab2a-4aa87e8b07c4)
![pic0](https://github.com/YimMenu/YimMenu/assets/136161998/7d85b383-224a-4ba3-a9f0-6ee1361a783a)

**Comment by Tayipc38** [comment]
> The new gui has certain problems, for example ...

The sliders are intentional. The wanted level is fixed in #1538 .
